src/client/components/SearchComponent/Search.js

```diff
@@ -0,0 +1,24 @@
+import React , {useState} from 'react';
+import search from '../../assets/images/SearchIcon.svg';
+import  './Search.styles.css';
+
+export default function Search() {
+    const [searchText, setSearchText] = useState(" ");
```

Comment:
I think it's better practice to have an empty string in the initial state, i.e `useState("")`.